code/__HELPERS/turfs.dm

```diff
@@ -1,7 +1,7 @@
 ///Returns location. Returns null if no location was found.
-/proc/get_teleport_loc(turf/location,mob/target,distance = 1, density = FALSE, errorx = 0, errory = 0, eoffsetx = 0, eoffsety = 0)
+/proc/get_teleport_loc(turf/location, mob/target, distance = 1, densitycheck = FALSE, closedturfcheck = FALSE, errorx = 0, errory = 0, eoffsetx = 0, eoffsety = 0)
```

Comment:
snake_case

density_check, not densitycheck
closed_turf_check, not closedturfcheck